Taskfile.yaml

```
@@ -63,3 +63,31 @@ tasks:
       - check-werf
     cmds:
       - werf build {{ .target }}
+  
+  lint:
+    desc: 'Run all linters in parallel. Important vars: "paths".'
```

```suggestion
    desc: 'Run all linters in parallel.'
```

"paths" not used here.